### PR TITLE
Release v1.14.0-beta3

### PR DESCRIPTION
**Pull request by GretaD**
Signed-off-by: greta <gretadoci@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.14.0-beta2 - 2022-09-05
+### Fixes
```
Comment (ChristophWurst) [inline CHANGELOG.md:5]:
```suggestion
### Fixed
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
At some point we should really consolidate the spread changelogs into *one* v1.14.0 entry

File: CHANGELOG.md (hunk 1 above)
Comment (GretaD) [reply CHANGELOG.md:5]: we use Fixes in other releases, and yes i can shoot a pr with one changelong for 1.14.0, later this week

Comment (ChristophWurst) [reply CHANGELOG.md:5]: https://github.com/nextcloud/mail/blame/main/CHANGELOG.md once.

https://keepachangelog.com/en/1.0.0/ for the specs.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.14.0-beta2 - 2022-09-05
```
Comment (ChristophWurst) [inline CHANGELOG.md:4]:
```suggestion
## 1.14.0-beta3 - 2022-09-05
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**